src/project.c

```diff
@@ -128,7 +134,21 @@ int count_sprites_per_view(uint32_t flags, uint8_t animation_type)
         }
 
     }
-    if (flags & CAR_IS_ANIMATED) { sprites_per_view = 4; }
+    if (flags & CAR_IS_ANIMATED) {
+        switch (animation_type)
+        {
+        case CAR_ANIMATION_NONE: return 1;
```

Comment:
are you sure it was meant to `return` from here? Seems quite different to what was done before

Reply:
None of the values modify any of the other values, so it has the same effective behavior. The function could be rewritten to use multiple returns with the same behavior.